<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="showModelStructureButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1. Show Model Structure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showTensorboardButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>2. Show TensorBoard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TestButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>3.Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DataArgumentButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>4.Data Augmantaion</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="imgIndex">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>25000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
